internal/flyerr/flyerr.go

```diff
@@ -90,6 +90,7 @@ func PrintCLIOutput(err error) {
 	suggestion := GetErrorSuggestion(err)
 
 	if description != "" {
+		fmt.Println()
 		fmt.Printf("\n%s", description)
```

Comment:
not sure what is happening here 🤔 
If you want an extra line, just prefix the string format with another `\n`, isn't it?

Reply:
yeah, that makes way more sense. I was following the pattern already existing in this file, but yes. you're correct. good catch.